elastic/kibana
# [Infra] Add metrics tab to the container views

Closes #180799 

## Summary

This PR adds the metrics tab to the container views - for now, the charts are the same as in the overview tab:
- Docker container 4 metrics - 4 charts
  ![image](https://github.com/elastic/kibana/assets/14139027/3e911d27-c268-4a36-8fd6-c35e989fd34b)

- Kubernetes container 2 metrics - 2 charts 
  <img width="1650" alt="image" src="https://github.com/elastic/kibana/assets/14139027/d626922c-2972-4858-abe5-9e1271bf2acd">

### Testing 
- Select Docker Containers from the inventory "Show" drop-down menu
  - Click on the metrics tab - the charts should be visible
  - Select a Docker container and click on "See all" for any metric inside the Metrics section
     - The metrics tab should be open and the correct metric in the left menu should be selected
  - Select a Kubernetes container and click on "See all" for any metric inside the Metrics section
     - The metrics tab should be open and the correct metric in the left menu should be selected


https://github.com/elastic/kibana/assets/14139027/684b4b60-6457-44aa-a1f2-ca0340049684

- Click on open as page link inside the flyout and the same steps should be checked for the page view (both Docker & Kubernetes)


https://github.com/elastic/kibana/assets/14139027/2f9e65be-e22d-4b38-9889-d997de20c081
